### ISSUE TITLE
How to specify local port for tcp protocol 

### ISSUE THREAD

**Issue by freemindLi** [traviscross/mtr#433]
Hey,We can see that mtr supports --localport parameter, but this parameter only works for UDP protocol. How can specify local port or not supported when using TCP protocol detection?

**Comment by wenlxie** [comment]
This local port parameter is only for UDP
https://github.com/traviscross/mtr/blob/master/ui/mtr.c#L112

**Comment by freemindLi** [comment]
@wenlxie  Yeah,I know this is just for UDP.I want to know whether this function can be supported by the tcp protocol?

**Comment by jimsiff2** [comment]
@rewolff fyi the [new MTR-like project ttl](https://github.com/lance0/ttl) has the ability to set a TCP source port, destination port, total number of flows (src/dst port pairs) to use for ECMP discovery, etc.  It would be great if MTR could support more flexibility with setting TCP source and destination ports.
